config/environments/development.rb

```diff
@@ -23,14 +23,21 @@
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
 
+    # Use Solid Cache for development testing
+    # :nocov:
```

Comment:
Question - I don't understand why we need to have  `# :nocov:` here. Do we not have a test for this scenario to have code coverage?